ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/ImageCvePage.tsx

```diff
@@ -63,8 +102,36 @@ function ImageCvePage() {
             </PageSection>
             <Divider component="div" />
             <PageSection className="pf-u-display-flex pf-u-flex-direction-column pf-u-flex-grow-1">
-                <WorkloadTableToolbar />
-                <Divider />
+                <div className="pf-u-background-color-100">
+                    <div className="pf-u-px-sm">
+                        <WorkloadTableToolbar />
+                    </div>
+                    <div className="pf-u-px-lg pf-u-pb-lg">
+                        {summaryRequest.error && (
+                            <Alert
```

Comment:
I used an error-style `Alert` here instead of the error empty state template, mostly because it is a smaller space and felt right. This is inconsistent with the error state in the header if the details query fails to load though...

I'll make a note to discuss this with UX at the next meeting too.